docs/source/tags/brush.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Brush
+type: tags
+order: 401
+is_new: t
+meta_title: Brush Tag for Image Segmentation Labeling
+meta_description: Customize Label Studio with brush tags for image segmentation labeling for machine learning and data science projects.
+---
+
+Use the Brush tag for image segmentation tasks where you want to apply a mask or use a brush to draw a region on the image.
```

Comment:
Enclose Brush with ** since it is a UI element. 
Note: All UI elements should be enclosed in **

Reply:
We don't do this in other tags, so if really need it it's better to update all the tags at once for consistency